backend/app/api/endpoints/cloud_projects_gitlab_mr.py

```
@@ -0,0 +1,64 @@
+    access = require_cloud_project_role(
+        db, project_id, current_user.id, BaseRole.Maintainer
+    )
+    mr_integration_service.enable(db, access.project, current_user.id)
+    db.commit()
+    return mr_integration_service.status(db, access.project)
+
+
+@router.get("/{project_id}/gitlab/mr-integration")
+def get_mr_integration_status(
+    project_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)